src/code.sh

```
@@ -4,10 +4,24 @@
 # and to output each line as it is executed -- useful for debugging
 set -e -x -o pipefail
 
-dx download "$input_bam" -o "$input_bam_name"
-dx download "$input_bam_index" -o "${input_bam_prefix}.bai"
+dx download "$input_file" -o "$input_file_name"
```

Shellcheck (0.10.0)

[warning] input_file is referenced but not assigned.

(SC2154)